### DISCUSSION TITLE
ACEBOTT QE007 Smart Home STEAM education kit could make a for a cool prototype?

### DISCUSSION THREAD

**Discussion by Hedda** [luar123/zigbee_esphome#1] [Ideas]
Could this ”ACEBOTT QE007” Smart Home STEAM education kit (made for kids) make a for a cool proof-of-concept prototype if paired with a ESP32-C6 development board instead?

https://www.cnx-software.com/2024/06/09/acebott-qe007-review-a-esp32-based-smart-home-steam-education-kit-for-8-years-old-kids/

**Comment by JochenFriedrich** [comment]
Yet another cool project is https://github.com/OLIMEX/ESP32-C6-EVB

Clones of the board can be found at aliexpress, also with a shell which can be mounted on a DIN rail.

A full featured Arduino example sketch with Router and zigbee OTA support is available at https://github.com/JochenFriedrich/ESP32-C6-EVB-Zigbee
